docs/tutorials/scheduled-workflows.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 18
+title: Scheduling Workflows
+description: Learn how to run DBOS workflows on a schedule.
+---
+
+In this guide, you'll learn how to execute DBOS workflows exactly once per time interval.
+
+### Defining A Workflow Function For Scheduling
+First, define your workflow.  Its parameters must be:
+- The `WorkflowContext`
+- The time that the run was scheduled (as a `Date`).
+- The time that the run was actually started (as a `Date`).  This can be used to tell if an exactly-once workflow was started behind schedule.
+
+```typescript
+import { Scheduled, Workflow, WorkflowContext } from '@dbos-inc/dbos-sdk';
+
+class ScheduledExample{
+  @Workflow()
+  static async scheduledFunc(ctxt: WorkflowContext, schedTime: Date, startTime: Date) {
+    ctxt.logger.info(`
+        Running a workflow every 5 seconds -
+          scheduled time: ${schedTime.toISOString()} /
+          actual start time: ${startTime.toISOString()}
+    `);
+  }
+}
+```
+
+### Scheduling The Workflow
+Then, annotate your method with a [`@Scheduled`](../api-reference/decorators.md#scheduled) decorator specifiying the schedule in a [`crontab`](https://en.wikipedia.org/wiki/Cron)-like format, which optionally specifies a constraint for seconds, followed by constraints for minutes, hours, day of month, month of year, and day of week.  For example, the following workflow will run every 30 seconds:
+
+```typescript
+import { Scheduled, Workflow, WorkflowContext } from '@dbos-inc/dbos-sdk';
+
+class ScheduledExample{
+  @Workflow()
+  @Scheduled({crontab: '*/30 * * * * *'})
+  static async scheduledFunc(ctxt: WorkflowContext, schedTime: Date, startTime: Date) {
+    ctxt.logger.info(``);
+  }
+}
+```
+
+The DBOS crontab format supports some common extensions, as seen in the following examples:
```

Comment:
Put a pointer to the full specification?

Reply:
I haven't found one.  I have one that I'm pasting in shortly.